doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -120,6 +121,7 @@ Architecture they must be implemented as per the following specifications:
 |`ra2.ntw.011`|NATless connectivity|When hosting workloads that require source and destination IP addresses to be preserved in the traffic headers, a CNI plugin that exposes the pod IP directly to the external networks (e.g. Calico, MACVLAN or IPVLAN CNI plugins) is required.|[req.inf.ntw.14](chapter02.md#23-kubernetes-architecture-requirements)|
 |`ra2.ntw.012`|Device Plugins|When hosting workloads matching the Network Intensive profile that require the use of FPGA, SR-IOV or other Acceleration Hardware, a Device Plugin for that FPGA or Acceleration Hardware must be used.|[e.cap.016](chapter02.md#221-cloud-infrastructure-software-profile-capabilities), [e.cap.013](chapter02.md#221-cloud-infrastructure-software-profile-capabilities)|
 |`ra2.ntw.013`|Dual stack CNI|The networking solution deployed within the implementation must use a CNI-conformant Network Plugin that is able to support dual-stack IPv4/IPv6 networking.|[req.inf.ntw.04](chapter02.md#23-kubernetes-architecture-requirements)|
+|`ra2.ntw.014`|Security Groups|The networking solution deployed within the implementation **must** support NetworkPolicy-es.|[infra.net.cfg.004](chapter02.md#23-kubernetes-architecture-requirements)|
```

Comment:
```suggestion
|`ra2.ntw.014`|Network Policy|The networking solution deployed within the implementation **must** support network policies.|[infra.net.cfg.004](chapter02.md#23-kubernetes-architecture-requirements)|
```

Reply:
In my head the Security Groups requirement is implemented by the NetworkPolicies.